packages/admin/admin/src/common/toolbar/ToolbarBreadcrumbs.tsx

```diff
@@ -0,0 +1,295 @@
+// TODO: Add theming and support for `sx` and `slotProps` when #1809 is merged: https://github.com/vivid-planet/comet/pull/1809
+import { ChevronDown, ChevronRight, ChevronUp } from "@comet/admin-icons";
+import {} from "@emotion/react";
+import { ButtonBase, css, ListItemText, Menu, MenuItem, styled, Theme, Typography } from "@mui/material";
+import React from "react";
+import { Link as RouterLink, LinkProps as RouterLinkProps } from "react-router-dom";
+
+import { useStackApi } from "../../stack/Api";
+import { useObservedWidth } from "../../stack/breadcrumbs/utils"; // TODO: Update import when merged: https://github.com/vivid-planet/comet/pull/1755
+
+// TODO: Remove debug code when we have stories for the new Toolbar to simulate a large stack
```

Comment:
Wouldn't it be easier to add a dev story right now?

Reply:
We'd have to export the component from the library for that.  
Since there is no use-case for the standalone breadcrumbs (it's only intended to be used in the new `Toolbar`) we decided against exporting it. 

I think we only need (dev-)stories for the full Toolbar anyway, we can add those to the feature-branch in a separate PR or when implementing the new Toolbar.

Reply:
We'll probably need an option to disable breadcrumbs then, but that's out of scope.